src/components/ticTacToeApp/index.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+
+import { utilis } from '../utilis'
+import './tic-tac-toe.css';
+import { soundGame, soundEndGame } from '../../config';
+import ListCells from '../ListCells';
+import logo from '../../logo.png';
+import ButtonTryAgan from '../ButtonTryAgain';
+import ButtonReset from '../ButtonReset';
+import ButtonUndo from '../ButtonUndo';
+import ModalWindow from '../StartMenu';
+
+class TicTacToeApp extends React.Component {
+    state = {
+        cells: utilis.createArray(9),
+        isNext: true,
+        winner: '',
+        isStart: true,
+        lastMove: null,
+        lastMoveComputer: null,
+        computerMode: false,
+        player: 'X',
+    }
+    handleClickCell = (i) => {
+        const cells = [...this.state.cells];
+        const winner = utilis.calculateWinner(this.state.cells);
```

Comment:
You can take all the values you need from this.state through destructuring.
For example: `const {cells} = this.state;`